### PR TITLE
Lecture 6 Notes

### PR DESCRIPTION
**Pull request by ncostar**

### CODE REVIEW

File: Lecture-6.ipynb [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+    "\n",
+    "### Passing Arrays to Methods\n",
+    "\n",
+    "#### Passing Entire Array\n",
+    "\n",
+    "To pass an array you specify the name of the array, as if it was a variable. The method will receive a copy of the objects reference. You can then manipulate the array within the scope of the method.\n",
+    "\n",
+    "#### Passing Array Elements to Methods\n",
+    "\n",
+    "To pass the element of an array simply use standard bracket notation to access the desired element. e.g `<array>[<size>]`.\n",
```
Comment (Liam-Scott-Russell) [inline Lecture-6.ipynb]: Is `<size>` the correct thing to include here, should it not be `<array>[<index>]`?

Comment (ncostar) [reply Lecture-6.ipynb]: Yep it's index whoops

File: Lecture-6.ipynb [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+    "\n",
+    "e.g `int[] ages = new int[10];` this will be indexed [0,9], with the default `int` values of `0`. I.e. `[0,0,0,0,0,0,0,0,0,0]`\n",
+    "\n",
+    "You can access elements of an array by their index. `<array name>[<element index>] = <value>`. Accessing elements via this method is a constant time operation.\n",
+    "\n",
+    "### Arguments for the method main\n",
+    "\n",
+    "When you run a program from the command line, all words after the class name will be passed to the main method in the args array.\n",
+    "\n",
+    "When you run the program (usually from the command line), you can pass args into the main method. This will be split into an array of strings that can be accessed through the `args` array.\n",
```
[the remark this answers is not in the document]
Comment (ncostar) [reply Lecture-6.ipynb:49]: Uhhh I `args[0]` is the first command line argument no?

[the remark this answers is not in the document]
Comment (Liam-Scott-Russell) [reply Lecture-6.ipynb:49]: Yiiikes NVM, Java is dumb and doesn't do 0 based sys parameters, ignore me

[the remark this answers is not in the document]
Comment (ncostar) [reply Lecture-6.ipynb:49]: lol oki

**Pull request merged**